ci/post/main.py

```diff
@@ -145,12 +155,29 @@ def main():
 		print("ERROR: Invalid release mode %s passed. Expected release or nightly!" % sys.argv[1])
 		sys.exit(1)
 
+	# bail if any build subflows failed
+	success = ((os.environ["LINUX_RESULT"] == "success") and (os.environ["WINDOWS_RESULT"] == "success"))	##!< true if both linux and windows builds successful
+	if not success:
+		print("ERROR: One or more builds failed to publish!")
+		sys.exit(1)
+	
+	# bail if tag_name doesn't match with sys.argv[1]
+	if (sys.argv[1] == "release") and (not tag_name.startswith("release_")):
+		print("ERROR: Invalid tag name detected. Expected release_ prefix for release builds but found %s (full ref = %s)" % tag_name, os.environ["GITHUB_REF"])
+		sys.exit(1)
+
+	if (sys.argv[1] == "nightly") and (not tag_name.startswith("nightly_")):
+		print("ERROR: Invalid tag name detected. Expected nightly_ prefix for nightly builds but found %s (full ref = %s)" % tag_name, os.environ["GITHUB_REF"])
+		sys.exit(1)
+
+
+	# Populate version and date
 	tag_name = os.environ["RELEASE_TAG"]	##!< commit tag string
 	date = datetime.now()	##!< current date
 	version = get_source_version(date.strftime(DATEFORMAT_VERSION), tag_name)	##!< form full version string
-	success = os.environ["LINUX_RESULT"] == "success" and os.environ["WINDOWS_RESULT"] == "success"	##!< true if both linux and windows builds successful
-
-	# check that tag_name is actually in the git repo and find the previous tag release
+	
+	# check that tag_name is in the
```

Comment:
Why is this comment incomplete?

Reply:
Indeed, why is that incomplete???  I might've been sidetracked right in the middle of updating it and forgot to save before committing the changes.